napari-workshops/notebooks/manual_annotation.md

```diff
@@ -88,10 +86,10 @@ Notice now how two new layers have been created, and that these layers have diff
 nbscreenshot(viewer)
 ```
 
-To add points you must enter add mode. This can be done by clicking on the add mode button in the top row of the control panel (2nd from the left, circle with a plus in it), or programatically from the notebook.
+To add points you must enter add mode. This can be done by clicking on the `add` mode button in the top row of the control panel (2nd from the left, circle with a plus in it), or programmatically from the notebook.
```

Comment:
```suggestion
To add points you must enter add mode. This can be done by clicking on the `add` mode button in the top row of the `layer controls` panel (2nd from the left, circle with a plus in it), or programmatically from the notebook.
```